ss13_tools/slur_detector/slur_file.py

```diff
@@ -0,0 +1,21 @@
+import os
+
+from colorama import Fore
+
+from .constants import SLURS_FILE
+
+if not os.path.exists(SLURS_FILE):
+    with open(SLURS_FILE, "w", encoding='utf-8') as f:
```

Comment:
Grr single-letter vars

Reply:
I always use f instead of file :pensive: